content/deploy/choose_cloud.md

```diff
@@ -0,0 +1,70 @@
+# Choosing where your JupyterHub will run
+
+## Should I deploy on my hardware or in the cloud?
+
+The first thing you'll decide is *where* your JupyterHub will run. The first
+question to ask is "will I run my JupyterHub on my own hardware, or will I
+run it in the cloud?" Running a JupyterHub on your own hardware is your best
```

Comment:
IMO, we shouldn't recommend this as a best choice. Running on your own hardware is going to be always problematic when stuff breaks. Here's a list of things that might happen to your hardware that are hard to fix without more engineering, while very easy to on the cloud:

1. You need more RAM / CPU
2. External public IPs (this is surprisingly hard with your own hardware)
3. Backups (snapshots are easy in all providers)
4. You screw up your box, and need to start over.

All these will probably happen at some point, and when most people think they have hardware support they're always going to underestimate these costs.

We should recommend they run on a cloud provider unless they can't, in which case they should run on their own hardware.